### ISSUE TITLE
Nothing comes up on the dropdown menus

### ISSUE THREAD

**Issue by Phoenix260** [jneilliii/OctoPrint-MyMiniFactory#14]
Hi,

I know I probably cause the error because I was trying to copy paste the QR code into the plugin from my phone late a night ... but I'm stuck ... and don't know what to do.

I've tried uninstalling, reinstalling, cleaning the plugin data ... 

I get this error in my logs:

```
2020-07-27 21:57:23,202 - octoprint.server.api.settings - ERROR - Could not load settings for plugin MyMiniFactory (0.1.4)
Traceback (most recent call last):
  File "/home/pi/oprint/local/lib/python2.7/site-packages/octoprint/server/api/settings.py", line 288, in _get_plugin_settings
    result = plugin.on_settings_load()
  File "/home/pi/oprint/local/lib/python2.7/site-packages/octoprint/plugin/types.py", line 1506, in on_settings_load
    if self.config_version_key in data:
TypeError: argument of type 'NoneType' is not iterable
2020-07-27 21:57:23,568 - octoprint.server.api.settings - ERROR - Could not load settings for plugin MyMiniFactory (0.1.4)
Traceback (most recent call last):
  File "/home/pi/oprint/local/lib/python2.7/site-packages/octoprint/server/api/settings.py", line 288, in _get_plugin_settings
    result = plugin.on_settings_load()
  File "/home/pi/oprint/local/lib/python2.7/site-packages/octoprint/plugin/types.py", line 1506, in on_settings_load
    if self.config_version_key in data:
TypeError: argument of type 'NoneType' is not iterable
```

I have no idea what it means ... I would typically go in to modify the python files, but I think I should leave these ones alone ...

I know it worked once, so how do I really delete the plugin to start over ... It seems to leave leftover data which doesn't get reset for me.
:(

**Comment by jneilliii** [comment]
You may have been hit by a bug here. SSH to your pi and run the command `nano ~/.octoprint/config.yaml` and then use the arrows on your keyboard to go down to the plugins > myminifactory section. I suspect next to it will be the word `null`. Delete the entire line and then ctrl+x on the keyboard and answer `y` to save the file and then enter on your keyboard to keep the same name.

**Comment by Phoenix260** [comment]
Thank you Thank you Thank you! that worked perfectly!

**Comment by jneilliii** [comment]
Cool, that bug is getting fixed in the next release of OctoPrint 1.4.1.
